VTUL/dlp-access
# Add a deployment instruction

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2415) (:star:)

# What does this Pull Request do? (:star:)
A deployment instruction for deploying dlp-access demo site. 

See preview: https://dev.d1ts8x5hxswi0x.amplifyapp.com/

# How should this be tested?
See https://github.com/VTUL/dlp-access/blob/LIBTD-2415/docs/deploy.md
This document is for a brand new deploy with all the demo records.

Please check wording and anything need to add. 

# Interested parties
Tag (@VTUL/dld-dev ) interested parties

(:star:) Required fields
